spring-boot-project/spring-boot-autoconfigure/src/main/java/org/springframework/boot/autoconfigure/elasticsearch/rest/RestClientProperties.java

```diff
@@ -47,6 +48,16 @@
 	 */
 	private String password;
 
+	/**
+	 * Connection timeout.
+	 */
+	private Duration connectionTimeout = Duration.ofSeconds(1);
+
+	/**
+	 * Read timeout.
+	 */
+	private Duration readTimeout = Duration.ofSeconds(30);
```

Comment:
Idem. -1 states "undefined" so I'd be curious to understand where are those 30 coming from?